CHANGELOG.md

```diff
@@ -5,6 +5,22 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [0.2.3] - 2020-08-19
+
+### Fixed
+
+-   ProvarDX runtests, metadatacache command fails in case we are having special characters in Scratch Org password
+
+### Changed
+
+-   ProvarDX trademark changes
+
+### Added
+
+-   Added .vscode folder containing
+    -   `launch.json`: debug configuration for vscode
+    -   `provardx-copyright.code-snippets`: used to add copyright tag on source files
+
```

Comment:
Do we need a changelog file? I find they're redundant if the git logs have proper commit messages in them and this is a public repo so anyone can see those.

Reply:
It's intended to be a curated, chronologically ordered list of notable changes for each version of the project and standard practice across all the Salesforce open source project on Github (which we're aligned to). It makes it easier for users and contributors to quickly see what notable changes have been made between each release (or version).